grammar.js

```diff
@@ -45,8 +45,13 @@ module.exports = grammar({
                 ),
             ),
         const: $ => seq('const', field('name', $.word), field('value', $._tc_expr)),
-        instruction: $ => seq(field('kind', $.word), choice(sep(',', $._expr), repeat($._tc_expr))),
-
+        instruction: $ => seq(
+          field('kind', $.word),
+          optional(choice(
+            $._expr,
```

Comment:
this looks like it breaks [TC assembly](https://turingcomplete.game/), as you just removed the reference to `_tc_expr`